packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -7,9 +7,11 @@
 /* eslint-disable no-alert */
 /* eslint-disable no-console */
 /* eslint-disable require-jsdoc */
+/* eslint-disable arrow-body-style */
```

Comment:
I hear all this talk about Prettier 🦄

Reply:
`arrow-body-style` is nice if you don't need to provide comments 😆